bunq/sdk/json/geolocation_adapter.py

```diff
@@ -0,0 +1,61 @@
+from bunq.sdk.json import converter
+from bunq.sdk.model.generated import object_
+
+
+class GeolocationAdapter(converter.JsonAdapter):
+    # Field constants
+    _FIELD_LATITUDE = 'latitude'
+    _FIELD_LONGITUDE = 'longitude'
+    _FIELD_ALTITUDE = 'altitude'
+    _FIELD_RADIUS = 'radius'
+
+    @classmethod
+    def can_deserialize(cls):
+        """
+        :rtype: bool
+        """
+
+        return False
+
+    @classmethod
+    def deserialize(cls, target_class, obj):
+        """
+        :type target_class: float|type
+        :type obj: dict
+
+        :raise: NotImplementedError
+        """
+
+        _ = target_class, obj
```

Comment:
seems rather useless?

Reply:
What would be the point of that? Should I just remove it?

See "Single Underscore: _" here: https://dbader.org/blog/meaning-of-underscores-in-python